packages/wallets/src/features/cashier/WalletCashier.scss

```diff
@@ -17,6 +17,8 @@
     padding: 4.8rem 1.6rem 0;
     background-color: #ffffff;
     overflow: scroll;
+    //prevent content bouncing on scroll on iPhone devices
+    overscroll-behavior: none;
```

Comment:
Will this stop header behaviour while scrolling the list?

Reply:
@nijil-deriv header behaviour will be the same